lib/jekyll/commands/serve.rb

```diff
@@ -2,6 +2,21 @@ module Jekyll
   module Commands
     class Serve < Command
       class << self
+        def process(opts)
```

Comment:
I couldn't find another way to bypass this [deletion](https://github.com/jekyll/jekyll/blob/master/lib/jekyll/configuration.rb#L157)